dcicutils/misc_utils.py

```diff
@@ -1349,6 +1349,25 @@ def capitalize1(s):
     return s[:1].upper() + s[1:]
 
 
+"""
+Python's UUID ignores all dashes, whereas Postgres is more strict
+http://www.postgresql.org/docs/9.2/static/datatype-uuid.html
+See also http://www.postgresql.org/docs/9.2/static/datatype-uuid.html
+And, anyway, this pattern is what our portals have been doing
+for quite a while, so it's the most stable choice for us now.
+"""
+
+uuid_re = re.compile(r'(?i)[{]?(?:[0-9a-f]{4}-?){8}[}]?')
```

Comment:
I am going to resolve this comment without taking it because my goal here was to be bug-for-bug compatible with what the portals use, so the support for this can be centralized. If there are any bugs in that, they can be addressed by a separate PR.